LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionProxy/TransparentProxyController.swift

```diff
@@ -168,37 +159,36 @@ public final class TransparentProxyController {
                 return internalManager
             }
 
-            if !triedLoadingManager {
-                triedLoadingManager = true
-
-                let manager = try? await NETransparentProxyManager.loadAllFromPreferences().first { manager in
-                    (manager.protocolConfiguration as? NETunnelProviderProtocol)?.providerBundleIdentifier == extensionID
-                }
-                self.internalManager = manager
+            let manager = try? await NETransparentProxyManager.loadAllFromPreferences().first { manager in
+                (manager.protocolConfiguration as? NETunnelProviderProtocol)?.providerBundleIdentifier == extensionID
             }
-
-            return internalManager
+            internalManager = manager
+            return manager
```

Comment:
I'm changing the code in this file to be more similar to code from `NetworkProtectionTunnelController` which is doing the same things for the VPN tunnel.  We had a bug here that was causing the configuration to be duplicated, likely caused by the `triedLoadingManager` class.